Bottom: b45b6238003702e7cea33d22cbd4a38e309aa743
Top:    94afe55da2afcbb058287b26e0054b7c10fb1d60
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 00:44:37 +0000

Refresh of extend-board-export

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 18424ed..bfa1e61 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -623,7 +623,6 @@ object3d_from_board_outline (void)
   poly_Free (&board_outline);
 
   return objects;
-#endif
 }
 
 void
